src/course-updates/course-handouts/CourseHandouts.scss

```diff
@@ -0,0 +1,16 @@
+.course-handouts {
+  .course-handouts-header {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: $spacer;
+
+    .course-handouts-header__title {
+      font-weight: 300;
```

Comment:
Please check if the font we are using is used for this header (Inter).

Reply:
Done

Reply:
Thanks 👍